Week2/MAKEME.md

```diff
@@ -36,6 +36,14 @@ console.log(doubleOddNumbers(myNumbers)); // ==> [2, 6]
 
 Rewrite the above `doubleOddNumbers` function using `map` and `filter`; don't forget to use `=>`.
 
+
+```js 
+const myNumbers = [1, 2, 3, 4];
+const newNumbers =[];
+
+myNumbers.filter(num => num%2 !=0).map(num => newNumbers.push(num*2));
```

Comment:
Looks good right?
Try to make it look even better by formatting like this, this way you can keep adding methods:
```
myNumbers 
    .filter
    .map
```